ecowitt2mqtt/data.py

```diff
@@ -91,6 +91,8 @@
 )
 
 DEFAULT_UNIQUE_ID = "default"
+DEFAULT_MODEL = "Unknown Model"
```

Comment:
Let's use `unknown_model` and `unknown_station_type` – that way, they style as constants and downstream libraries can implement whatever human-friendly language they want.

(Your solution can totally do the same thing; just wanting to keep the practice of not human-friendly-ifying until downstream.)

Reply:
Done!